React/CoreModules/RCTAlertController.m

```diff
@@ -20,7 +20,23 @@ @implementation RCTAlertController
 - (UIWindow *)alertWindow
 {
   if (_alertWindow == nil) {
-    _alertWindow = [[UIWindow alloc] initWithFrame:RCTSharedApplication().keyWindow.bounds];
+
+#if defined(__IPHONE_OS_VERSION_MAX_ALLOWED) && defined(__IPHONE_13_0) && \
+    __IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_13_0
```

Comment:
Is this #if actually required? It looks like that the condition is the same as `if (@available(iOS 13.0, *))`

Reply:
Yes they are redundant. I'll remove these lines